src/core.jsx

```diff
@@ -52,6 +53,11 @@ export default (
         return !!print(this.props, this.context)
       }
 
+      if (print === undefined) {
```

Comment:
And maybe adding a comment on top of that code bloc to explain the purpose of it